scripts/us_census/geojsons_low_res/plotter.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Plots and compares two Geojson files: before and after running simplify.py.
+
+    Typical usage:
+    python3 plotter.py --original_path original-data/geoId-01.geojson
+                       --simplified_path simplified-data/geoId-01-simple.geojson
+"""
+
+from absl import flags
+from absl import app
+import geopandas as gpd
+import matplotlib
+matplotlib.use('TkAgg')
+import matplotlib.pyplot as plt
+
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('original_path', default=None,
+                    help='Path to original geojson to be compared.')
+flags.DEFINE_string('simplified_path', default=None,
+                    help='Path to simplified geojson to be compared.')
+flags.mark_flag_as_required('original_path')
+flags.mark_flag_as_required('simplified_path')
+
+_, (ax1, ax2) = plt.subplots(ncols=2, sharex=True, sharey=True)
```

Comment:
Let's put this inside `compare_plots`, avoiding global variables.

Reply:
This actually needs to be out here given the way the code is written now. If you want to put many geojsons in one plot, you need one common set of axes to draw everything on. Pushing this into `compare_plots` would make the code in `generate_mcf.py` more involved, as it would have to explicitly deal with the axis objects itself, passing them into `compare_plots` as parameters.